src/AppBundle/Controller/PostController.php

```diff
@@ -30,4 +30,16 @@ public function onlPostViewAction($postId)
     {
 
     }
+
+    public function testJenkinsCI()
+    {
+        $string = 'string';
+
+        if ($string){
+            $style = 'bad';
+        return false;
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: PEAR.WhiteSpace.ScopeIndent.Incorrect
**Severity**: ERROR
**File**: /var/lib/jenkins/workspace/test2/src/AppBundle/Controller/PostController.php L40

Line indented incorrectly; expected at least 12 spaces, found 8


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1969832011>*